app/javascript/components/ansible-playbook-edit-catalog-form/index.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from 'react';
+import MiqFormRenderer from '@@ddf';
+import createSchema from './schema';
+
+const AnsiblePlayBookEditCatalogForm = ({ initialData }) => {
+    const [data, setData] = useState({ isLoading: true, formData: {} , currencies: []});
+    useEffect(() => {
+        const apiCall1 = API.get(`/api/service_templates/${initialData.catalogFormId}`);
+        const apiCall2 = API.get('/api/currencies/?expand=resources&attributes=id,full_name,symbol,code&sort_by=full_name&sort_order=ascending')
```

Comment:
lets provide a meaningful name instead of `apiCall1` and `apiCall2`

Reply:
made the changes.Thanks